lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1663,7 +1663,9 @@ SwiftExpressionParser::Parse(DiagnosticManager &diagnostic_manager,
   // Helper function to diagnose errors in m_swift_scratch_context.
   unsigned buffer_id = UINT32_MAX;
   auto DiagnoseSwiftASTContextError = [&]() {
-    assert(m_swift_ast_ctx.HasErrors() && "error expected");
+    assert((m_swift_ast_ctx.HasErrors() ||
```

Comment:
Would it be useful to have a method `HasAnyErrors` in `SwiftASTContext` which returns `HasErrors() || HasClangImporterErrors()` ?

Reply:
I thought about that, but I decided that it's better to be explicit about wanting to include ClangImporter errors at the call site.